src/FileService.Models.Broker/Responses/IAddImageResponse.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+
+namespace LT.DigitalOffice.Broker.Responses
+{
+    public interface IAddImageResponse
+    {
+        Guid Id { get; }
+
+        static object CreateObj(Guid id)
+        {
+            return new
+            {
+                Id = id
+            };
+        }
+    }
+}
```

Comment:
If simple use` _requestClient.GetResponse<IOperationResult<Guid>>`?

Reply:
I guess it is not bad, plus now MessageService wait for this response

Reply:
I also have assumptions that in the future, not only the id will be returned, but also a small image (for example)

Reply:
Why something need return besides Image id?

Reply:
We talk about it on Sunday we talk about what we have to return after creating. Now Create a workspace for example returns the only guide, but I guess in future it will return base info about new workspace - id, name, small image

Reply:
of course, we can return only guid and ask front to send a new request for more info of entity, but is it right?

Reply:
We talk about it on Sunday, that when created entity, we will be returns Only base information about entity, name and small image - addition information